### PR TITLE
RND203-322: Add AR_SREV_9271 macro

### PR DESCRIPTION
**Pull request by Rafal-Litwinski-Mobica**

### CODE REVIEW

**Review by Mikolaj-Filar-Mobica — APPROVED** [review]

File: sys/dev/ath/ath_hal/ar5416/ar5416reg.h [hunk 1]
```diff
@@ -807,7 +808,9 @@
 	(AR_SREV_KIWI(_ah) && \
 	 AH_PRIVATE((_ah))->ah_macRev >= AR_XSREV_REVISION_KIWI_13)
 
-/* Not yet implemented chips */
-#define	AR_SREV_9271(_ah)	0
+/* K2 (9271) */
+
+#define	AR_SREV_9271(_ah)
```
Comment (Mikolaj-Filar-Mobica) [inline sys/dev/ath/ath_hal/ar5416/ar5416reg.h]: Missing `\` at the end of line

**Review by Mikolaj-Filar-Mobica — CHANGES_REQUESTED** [review]
Missing \ at the end of line

File: sys/dev/ath/ath_hal/ar5416/ar5416reg.h [hunk 2]
```diff
@@ -698,6 +698,7 @@
 #define	AR_XSREV_REVISION_KIWI_11	1	/* Kiwi 1.1 */
 #define	AR_XSREV_REVISION_KIWI_12	2	/* Kiwi 1.2 */
 #define	AR_XSREV_REVISION_KIWI_13	3	/* Kiwi 1.3 */
+#define AR_XSREV_VERSION_K2         0x140	/* K2 (AR9160) */
```
Comment (Mikolaj-Filar-Mobica) [inline sys/dev/ath/ath_hal/ar5416/ar5416reg.h]: AR9160?

File: sys/dev/ath/ath_hal/ar5416/ar5416reg.h (hunk 1 above)
Comment (Rafal-Litwinski-Mobica) [reply sys/dev/ath/ath_hal/ar5416/ar5416reg.h]: fixed

File: sys/dev/ath/ath_hal/ar5416/ar5416reg.h (hunk 2 above)
Comment (Rafal-Litwinski-Mobica) [reply sys/dev/ath/ath_hal/ar5416/ar5416reg.h]: fixed

**Review by Mikolaj-Filar-Mobica — APPROVED** [review]

**Review by nzulinski-mobica — APPROVED** [review]

**Pull request merged**